app/nodejs-v10/src/test/scala/io/scalajs/nodejs/fs/FsClassesTest.scala

```diff
@@ -10,17 +10,17 @@ class FsClassesTest extends AnyFunSpec {
 
   describe("ReadStream") {
     it("supports constructor(") {
-      assert(new ReadStream("package.json") !== null)
+      assert(new ReadStream("package.json").readableLength > 0)
       assert(new ReadStream(Buffer.from("package.json")) !== null)
       assert(new ReadStream(new URL(s"file:///${dirname}/package.json")) !== null)
     }
   }
 
   describe("WriteStream") {
     it("supports constructor") {
-      assert(new WriteStream("package.json") !== null)
-      assert(new WriteStream(Buffer.from("package.json")) !== null)
-      assert(new WriteStream(new URL(s"file:///${dirname}/package.json")) !== null)
+      assert(new WriteStream("NO_SUCH_FILE").writableLength === 0)
```

Comment:
Opening with correct `WriteStream` empty its file, so changed the file to be opened.